ocs_ci/deployment/deployment.py

```diff
@@ -222,14 +222,23 @@ def deploy_ocs(self):
         assert ceph_health_check(
             namespace=config.ENV_DATA['cluster_namespace']
         )
-        # patch gp2 (EBS) storage class as 'non-default'
-        logger.info("Patch gp2 storageclass as non-default")
-        patch = " '{\"metadata\": {\"annotations\":{\"storageclass.kubernetes.io/is-default-class\":\"false\"}}}' "
-        run_cmd(
-            f"oc patch storageclass gp2 "
-            f"-p {patch} "
-            f"--request-timeout=120s"
-        )
+        # patch gp2/thin storage class as 'non-default'
+        sc_to_patch = None
+        platform = config.ENV_DATA.get('platform')
+        if platform in ('aws', 'AWS'):
```

Comment:
please create constants for AWS and Vsphere

Reply:
Done